engine/runtime-parser-dsl/src/main/java/org/enso/runtime/parser/dsl/IRChild.java

```diff
@@ -0,0 +1,19 @@
+package org.enso.runtime.parser.dsl;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Constructor parameter annotated with this annotation will be represented as a child field in the
+ * generated super class. Children of IR elements for a tree. A child will be part of the methods
+ * traversing the tree, like {@code mapExpression} and {@code children}. The parameter type must be
+ * a subtype of {@code org.enso.compiler.ir.IR}.
```

Comment:
```suggestion
 * a subtype of {@link org.enso.compiler.ir.IR}.
```

Reply:
Cannot use `link` here, because `runtime-parser-dsl` project does not depend on `runtime-parser`.